src/modules/Dashboard/features/Applet/AddUser/LinkGenerator/LinkGenerator.types.ts

```diff
@@ -4,6 +4,6 @@ export type InviteLink = {
 };
 
 export type LinkGeneratorProps = {
-  setInviteLink: (val: InviteLink | null) => void;
+  setInviteLink: (link: InviteLink | null) => void;
```

Comment:
Maybe remove possibility to invite link be null, because of this check?
![image](https://github.com/ChildMindInstitute/mindlogger-admin/assets/116339462/f836d45c-f779-4549-a270-514b6a9a493e)
So we don't render LinkForm component if inviteLink is undefined or null.